File: frontend/src/lib/brand.ts
// Product brand name — the SPA's single config point for a rebrand
// (backend twin: api/Core/Brand.py, which documents the remaining
// static checklist: index.html meta, PWA manifest, Privacy legal
// text, plain-text email bodies).
//
// Override at build time with VITE_BRAND_NAME.
export const BRAND_NAME =
  (import.meta.env.VITE_BRAND_NAME as string | undefined) ?? "DineroBook";

File: api/Core/Brand.py
"""Product brand name — the single config point for a rebrand.

"DineroBook" reads MSB-flavored; the pivot (HANDOFF.md §2) makes a
rename plausible. Everything runtime-variable reads the name from
here (backend) or ``frontend/src/lib/brand.ts`` (SPA), so a rebrand
is an env change plus the small static checklist below — not a
codebase-wide hunt.

Static spots a rebrand still touches by hand:
  * ``frontend/index.html`` — <title> + meta/OG tags
  * PWA manifest + icons
  * ``frontend/src/routes/Privacy.tsx`` — legal text (needs legal
    review at rebrand time anyway)
  * plain-text email body constants (grep for the old name)
  * the custom domain + ``APP_BASE_URL`` / ``WEBAUTHN_RP_ID`` env
"""
import os


def get_brand_name() -> str:
    return os.environ.get("BRAND_NAME", "DineroBook")
